### ISSUE TITLE
Firefox 69 not working 

### ISSUE THREAD

**Issue by donaghq** [darktrojan/openwith#247]
Hello

This is working well in Chrome but has recently stopped working in latest version of FF. The installation works and it can find one browser but it does not give the open to use it.

![image](https://user-images.githubusercontent.com/38651766/65036770-17189000-d944-11e9-8275-e6ee9a943236.png)

This icon never becomes functional even after finding the browser

![image](https://user-images.githubusercontent.com/38651766/65036742-07994700-d944-11e9-913b-2c3b2f0bca94.png)

This is what it looks like in Chrome

![image](https://user-images.githubusercontent.com/38651766/65036822-3f07f380-d944-11e9-917c-efac045e0feb.png)

Thanks

DQ

**Comment by Sopor** [comment]
For me it stopped worked when i updated from 70 to 70.0.1 and it seems that a whole lot of other extension have also stopped working.

**Comment by donaghq** [comment]
yeah dashlane has stopped working for me too, i stopped using firefox

thanks

DQ

**Comment by Sopor** [comment]
It seems to be a broken profile for me. I used an older backup and everything seems to work now with 70.0.1.

**Comment by donaghq** [comment]
I did not backup.
